cunumeric/args.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021-2022 NVIDIA Corporation
```

Comment:
I think for new files we want the copyright to start on the current year.

Reply:
Pretty sure some other OSS projects I work on get this entirely wrong, old habits...